WordPress/src/main/java/org/wordpress/android/ui/notifications/blocks/CommentUserNoteBlock.java

```diff
@@ -79,7 +79,7 @@ public View configureView(View view) {
                 noteBlockHolder.avatarImageView.setOnTouchListener(null);
             }
         } else {
-            noteBlockHolder.avatarImageView.setImageResource(R.drawable.gravatar_placeholder);
+            noteBlockHolder.avatarImageView.showDefaultGravatarImage();
             noteBlockHolder.avatarImageView.setOnTouchListener(null);
         }
 
```

Comment:
Just an FYI that I tested this and the `UserNoteBlock` change below and they're fine.